server/database/fakeData/index.ts

```diff
@@ -0,0 +1,48 @@
+import sequelize from '../config/connection';
+// eslint-disable-next-line import/extensions
+import * as data from './data.json';
+
+import {
+  Campaign,
+  Donor,
+  Family,
+  Donation,
+  Contact,
+  Category,
+  Capon,
+} from '../models';
+
+const {
+  campaigns, capons, categories, messages, donations, donors, families,
+} = data;
+
+const buildData = async () => {
```

Comment:
I think it's better to rename the file name to something that shows that we are building the fake data here.